app/upgrades/v15/upgrades.go

```diff
@@ -286,3 +296,12 @@ func registerOsmoIonMetadata(ctx sdk.Context, bankKeeper bankkeeper.Keeper) {
 	bankKeeper.SetDenomMetaData(ctx, uosmoMetadata)
 	bankKeeper.SetDenomMetaData(ctx, uionMetadata)
 }
+
+func removeExitFee(ctx sdk.Context, gammKeeper gammkeeper.Keeper, poolsWithExitFee []uint64) {
+	for _, poolId := range poolsWithExitFee {
+		err := gammKeeper.DeletePool(ctx, poolId)
```

Comment:
Why are we deleting these pools? The pools should stay, only the exit fee setting is removed